cupy/random/generator.py

```diff
@@ -135,6 +135,16 @@ def dirichlet(self, alpha, size=None, dtype=float):
         self.rk_seed += cupy.core.internal.prod(size)
         return y
 
+    def exponential(self, scale=1.0, size=None, dtype=float):
+        """Returns an array of samples drawn from a exponential distribution.
+
+        .. seealso::
+            :func:`cupy.random.exponential` for full documentation,
+            :meth:`numpy.random.RandomState.exponential`
+        """
+        x = self.standard_exponential(size, dtype)
```

Comment:
Update `size` using `scale.shape` before calling `standard_exponential`.  The following is failing.
```
cupy.random.exponential(cupy.array([1., 2.]), size=None)
```